ros/lib/models.py

```diff
@@ -28,6 +28,17 @@ def display_performance_score(self):
 
         return display_performance_score
 
+    @property
+    def idling_time(self):
+        idling_percent = (self.performance_record['kernel.all.cpu.idle']
+                          / self.performance_record['total_cpus']) * 100
+        # idling % of 24 hours and converting hours into mins
+        return 24 * (idling_percent / 100) * 60
```

Comment:
What is unit of `self.performance_record['kernel.all.cpu.idle']`?

Reply:
As discussed, modifying it to just %